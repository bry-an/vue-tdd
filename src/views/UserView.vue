<template>
    <div>
        <v-user-profile 
            :user="user" />
        <v-user-search-form />
    </div>
</template>

<script>
import VUserProfile from "@/components/VUserProfile"
import VUserSearchForm from "@/components/VUserSearchForm"
export default {
    name: "UserView", 
    components: {
        VUserProfile, 
        VUserSearchForm
    }
}
</script>