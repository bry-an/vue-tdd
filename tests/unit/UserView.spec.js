import { shallowMount } from "@vue/test-utils"
import UserView from "@/views/UserView"

describe("UserView", () => {
    it("renders the component", () => {})
    //arrange
    const wrapper = shallowMount(UserView) // render only the first level of dependencies

    // assert
    expect(wrapper.html()).toMatchSnapshot()
})