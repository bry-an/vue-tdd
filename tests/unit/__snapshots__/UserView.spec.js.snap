// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserView renders the component 1`] = `
<div>
  <v-user-profile-stub user="[object Object]"></v-user-profile-stub>
  <v-user-search-form-stub></v-user-search-form-stub>
</div>
`;
